admin/addField.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from "react";
+import * as firebase from 'firebase/app';
+
+export const AddField = () => {
+    const [field, setField] = useState('');
+
+    return <form onSubmit={e => addFieldToLinksInDatabase(e, field)}>
+        <label><h3>New field title</h3></label>
+        <input value={field} type="text" onChange={e => setField(e.target.value)}/>
+        <input type="submit" value="Submit" />
```

Comment:
To ting:

- `<input type="text" />` er default-oppførselen - så du kan bare droppe å ha det med om du vil. Dvs `<input />`.
- Hvis du vil ha en submit-knapp, anbefaler jeg å bruke `<button>Klikk for å legge til</button>` istedenfor. Det fikser noen quirky greier i visse browsers, og er mer hva man er "vant til å se" rundt om. Det fungerer ellers helt likt som `<input type="submit" />` altså, så du kan jo velge selv.